sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2306,6 +2306,28 @@ def deinit(self):
 
         del globals()['platform_chassis']
 
+    def load_feature_flags(self):
+        """
+        Load feature enable/skip flags from platform files.
+        """
+        platform_pmon_ctrl_file_path = self.get_platform_pmon_ctrl_file_path()
+        if platform_pmon_ctrl_file_path is not None:
+            # Load the JSON file
+            with open(platform_pmon_ctrl_file_path) as file:
+                data = json.load(file)
+                if ENABLE_SFF_MGR_FLAG_NAME in data:
+                    enable_sff_mgr = data[ENABLE_SFF_MGR_FLAG_NAME]
+                    self.enable_sff_mgr = isinstance(enable_sff_mgr, bool) and enable_sff_mgr
+
+    def get_platform_pmon_ctrl_file_path(self):
+        """
+        Get the platform PMON control file path.
+
+        Returns:
+            str: The platform PMON control file path.
```

Comment:
@longhuan-cisco we don't need to repeat code for parsing again. reuse the existing code hence the reason i suggested to use the same approach alreay used to skip CMIS mgr.We can revisit this later for future enhancement for both cmis and sff mgr or any new addition

Reply:
@longhuan-cisco to keep the code clean and to NOT repeat the parsing code for every newly introduced parameter to xcvrd daemon, suggest to follow the same approach as skip_cmis_mgr. It just unnecessary to duplicate code for every new parameter introduced here and future

Reply:
updated to use the old approach of going via docker-pmon.supervisord.conf, corresponding PR was raised https://github.com/sonic-net/sonic-buildimage/pull/18054